safed.dd

```diff
@@ -160,7 +160,7 @@ std:cout &lt;&lt; "Hello World!" &lt;&lt; std::endl;	)
 	)
 
 	$(P
-	Whereas pointers were important in C, C++ embraced them as the main vehicle for the Standard Library. STL algorithms use iterators, objects that are either pointers themselves or imitate the behavior (and the pitfalls) of pointers. Just like with pointers, a programmer's error in using iterators leads to undefined behavior (see the $(CODE swap_range) example).
+	Whereas pointers were important in C, C++ embraced them as the main vehicle for the Standard Library. STL algorithms use iterators, objects that are either pointers themselves or imitate the behavior (and the pitfalls) of pointers. Just like with pointers, a programmer's error in using iterators leads to undefined behavior (see the $(CODE swap_ranges) example).
```

Comment:
The example he's referring to is in paragraph 6 that reads:

"One notorious example is the std::swap_ranges algorithm, which takes three iterators."